tests/CMakeLists.txt

```diff
@@ -50,6 +50,10 @@ if (BUILD_YICES2)
   add_subdirectory(yices2)
 endif()
 
+if (${BUILD_YICES2} AND ${BUILD_BTOR} AND ${BUILD_MSAT} AND ${BUILD_CVC4})
```

Comment:
we will build portfolio only if all the solvers are available. Is it intended?

Reply:
This is building the portfolio solver test that uses all four solvers.

Reply:
oh, i missed that. then it is fine